src/main/java/org/fundacionjala/coding/cynthia/ShortInner.java

```diff
@@ -0,0 +1,52 @@
+package org.fundacionjala.coding.cynthia;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Created by CynthiaTerrazas on 5/30/2017.
+ */
+public class ShortInner {
+    /**
+     * this method build the correct sentence.
+     *
+     * @param cadena is the enter to the method.
+     * @return cadenaTotal is the final result.
+     */
+    public String result(String cadena) {
+        StringBuilder cadenaTotal = new StringBuilder();
+        String[] arrayCadena = cadena.split(" ");
+        for (String word : arrayCadena) {
+            cadenaTotal.append(" ");
```

Comment:
Instead of adding blank spaces in the stringbuilder
you can use a StringJoiner cadenaTotal = new StringJoiner(" ");